Makefile

```diff
@@ -99,6 +103,11 @@ gosec: scripts/go/bin/gosec
 		-conf=./scripts/go/configs/gosec.json \
 		$(GO_FILES)
 
+golangci-lint: scripts/go/bin/golangci-lint
```

Comment:
I'd suggest replacing the existing `lint-go` target

Reply:
Yes!